[2023-06-30T13:10:52.875+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: order-letter_NER.post_to_dynamics_task manual__2023-06-30T12:49:03.146226+00:00 [queued]>
[2023-06-30T13:10:52.889+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: order-letter_NER.post_to_dynamics_task manual__2023-06-30T12:49:03.146226+00:00 [queued]>
[2023-06-30T13:10:52.889+0000] {taskinstance.py:1308} INFO - Starting attempt 3 of 6
[2023-06-30T13:10:52.908+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): post_to_dynamics_task> on 2023-06-30 12:49:03.146226+00:00
[2023-06-30T13:10:52.915+0000] {standard_task_runner.py:57} INFO - Started process 18124 to run task
[2023-06-30T13:10:52.919+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'order-letter_NER', 'post_to_dynamics_task', 'manual__2023-06-30T12:49:03.146226+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/order-letter-extraction.py', '--cfg-path', '/tmp/tmpdvidtlq_']
[2023-06-30T13:10:52.922+0000] {standard_task_runner.py:85} INFO - Job 113: Subtask post_to_dynamics_task
[2023-06-30T13:10:52.987+0000] {task_command.py:410} INFO - Running <TaskInstance: order-letter_NER.post_to_dynamics_task manual__2023-06-30T12:49:03.146226+00:00 [running]> on host f7b244b51c6e
[2023-06-30T13:10:53.226+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abin Jilson' AIRFLOW_CTX_DAG_ID='order-letter_NER' AIRFLOW_CTX_TASK_ID='post_to_dynamics_task' AIRFLOW_CTX_EXECUTION_DATE='2023-06-30T12:49:03.146226+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-30T12:49:03.146226+00:00'
[2023-06-30T13:10:53.237+0000] {logging_mixin.py:149} INFO - ['AMSTERDAM Lamp', 'TOKYO Guest Chair,\r\nblue', 'ROME Guest Chair, green', 'Conference Bundle 1-6', 'ANTWERP\r\nConference Table.']
[2023-06-30T13:10:53.238+0000] {logging_mixin.py:149} INFO - [11, 6, 13, 2, 2]
[2023-06-30T13:10:56.760+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/order-letter-extraction.py", line 174, in dynamics_upload_main
    "quantity": quantity_list[i],
IndexError: string index out of range
[2023-06-30T13:10:56.776+0000] {taskinstance.py:1345} INFO - Marking task as UP_FOR_RETRY. dag_id=order-letter_NER, task_id=post_to_dynamics_task, execution_date=20230630T124903, start_date=20230630T131052, end_date=20230630T131056
[2023-06-30T13:10:56.809+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 113 for task post_to_dynamics_task (string index out of range; 18124)
[2023-06-30T13:10:56.833+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-30T13:10:56.863+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
